composables/data/productBanners.ts

```diff
@@ -0,0 +1,22 @@
+import { ProductBanner } from './types'
+
+export async function findProductBanners(id?: number) {
+    const config = useRuntimeConfig()
+    id = id ?? config.systemMessagesId
+    return await useAviary(`/system_messages/${id}`)
+}
+
+export function normalizeFindProductBannersResponse(productBannersData: Record<string, any>): ProductBanner[] {
+    return productBannersData.value?.productBanners?.map(normalizeProductBanner) ?? []
+}
+
+export function normalizeProductBanner(banner: Record<string, any>): ProductBanner {
```

Comment:
This could be simplified to something like ` => ({...banner.value, frequencyInHours: banner.value.frequency});`. Or, if you're ok with keeping the property as `frequency` instead of `frequencyInHours`, you could just do `=> ({...banner.value});`

Reply:
As above. Want to specifically define the return object here independent from what the json in the API says.